app/src/main/java/org/dhis2/data/dhislogic/DhisProgramUtils.kt

```diff
@@ -161,7 +161,7 @@ class DhisProgramUtils @Inject constructor(val d2: D2) {
             ProgramType.WITH_REGISTRATION -> {
                 program.trackedEntityType()?.displayName() ?: defaultTrackerLabel
             }
-            ProgramType.WITHOUT_REGISTRATION -> defaultEventLabel
+            ProgramType.WITHOUT_REGISTRATION -> program.eventLabel() ?: defaultEventLabel
```

Comment:
isn't this method deprecated? aren't they changing to `program.displayEventLabel()`?

Reply:
You are rigth, in fact this doesn't came here, I forgot to remove it. Thanks